distr_tracing/distr_tracing_rn/distr-tracing-rn-3-1.adoc

```diff
@@ -0,0 +1,108 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="distributed-tracing-rn-3-1"]
+= Release notes for {DTProductName} 3.1
+:context: distributed-tracing-rn-3-1
+
+toc::[]
+
+include::modules/distr-tracing-product-overview.adoc[leveloffset=+1]
```

Comment:
IMO, it would make more sense to make this module a simple snippet, with no title etc, so that there aren't 2 headings together with no text between when it is included in these assemblies. Wdyt?

Reply:
Thank you for the suggestion. I am adding it to the list of improvements for the next release because there is currently no page that presents an overview and introduction into this product, so I plan to create such a page for the next release. Once I have that figured out, I'll revisit this section.